src/constants/index.ts

```diff
@@ -327,6 +329,26 @@ export const SWAP_CONTRACT_GAS_ESTIMATES_MAP = {
   removeLiquidityOneToken: BigNumber.from("250000"), // 232947
 }
 
+export interface WalletInfo {
+  connector?: AbstractConnector
+  name: string
+}
+
+export const SUPPORTED_WALLETS: { [key: string]: WalletInfo } = {
+  METAMASK: {
+    connector: injected,
+    name: "MetaMask",
+  },
+  WALLET_CONNECT: {
+    connector: walletconnect,
+    name: "WalletConnect",
+  },
+  WALLET_LINK: {
+    connector: walletlink,
+    name: "Coinbase Wallet",
+  },
+}
+
```

Comment:
this is duplicating the `wallets` array here: https://github.com/saddle-finance/saddle-frontend/blob/master/src/components/ConnectWallet.tsx#L13

either use that here or replace those references with `SUPPORTED_WALLETS`